homeassistant/components/schlage/__init__.py

```diff
@@ -32,7 +29,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: SchlageConfigEntry) -> b
     except WarrantException as ex:
         raise ConfigEntryAuthFailed from ex
 
-    coordinator = SchlageDataUpdateCoordinator(hass, username, pyschlage.Schlage(auth))
+    coordinator = SchlageDataUpdateCoordinator(
+        hass, entry, username, pyschlage.Schlage(auth)
+    )
```

Comment:
We could derive the username from the entry data in the coordinator.

Reply:
I kept it, since the `username` is not exclusively created for the coordinator (_it is also used above_)